code/modules/client/preferences.dm

```diff
@@ -436,6 +436,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			.++
 
 /datum/preferences/proc/validate_quirks()
+	if(CONFIG_GET(flag/disable_quirk_points))
+		return
```

Comment:
you still need to check for quirks which break selection rules

Reply:
This proc is not responsible for such functionality, and only checks to ensure that the quirk balance is not a positive number.